config/l5-swagger.php

```diff
@@ -82,6 +82,16 @@
 
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Middleware allows to prevent unexpected access to API documentation
+    |--------------------------------------------------------------------------
+     */
+    'middleware' => [
+        'api' => ['web'],
```

Comment:
this is middleware group from Laravel
https://laravel.com/docs/5.3/middleware#middleware-groups